src/main/java/fr/cnes/sonar/plugins/icode/check/ICodeSensor.java

```diff
@@ -16,6 +16,7 @@
  */
 package fr.cnes.sonar.plugins.icode.check;
 
+import fr.cnes.sonar.plugins.icode.ICodePlugin;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'fr.cnes.sonar.plugins.icode.ICodePlugin'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonaricode&pullRequest=31&issues=AWXTgLl8ashydt7kJTGm&open=AWXTgLl8ashydt7kJTGm)

[ISSUE_KEY]: AWXTgLl8ashydt7kJTGm